modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java

```diff
@@ -361,7 +363,25 @@ public String getRedirectURL() throws PortalException {
 		return ddmFormInstanceSettings.redirectURL();
 	}
 
-	public String getSubmitLabel() throws PortalException {
+	public String getSubmitLabel() throws JSONException, PortalException {
+		DDMFormInstance ddmFormInstance = getFormInstance();
+
+		if (ddmFormInstance == null) {
+			return null;
```

Comment:
@brunofariasdeo, shouldn't we return a blank String instead? In this case, I think it's less problematic return an empty String instead of a null pointer, what do you think?